pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -468,6 +472,7 @@ func (r *ReconcileLogStorage) Reconcile(ctx context.Context, request reconcile.R
 			hdler,
 			reqLogger,
 			ctx,
+			install.ControlPlaneReplicas,
```

Comment:
Since ESGateway is already passed the `install`, there is no need to separately pass ControlPlaneReplicas.